.ci/ci_check.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+
+LOG_ERROR()
+{
+    local content=${1}
+    echo -e "\033[31m[ERROR] ${content}\033[0m"
+    exit 1
+}
+
+execute_cmd()
+{
+    command="${1}"
+    #LOG_INFO "RUN: ${command}"
+    eval ${command}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Double quote to prevent globbing and word splitting.](https://app.codacy.com/gh/FISCO-BCOS/go-sdk/pullRequest?prid=5569463)